spec/lib/scrabble_spec.rb

```diff
@@ -0,0 +1,50 @@
+require './lib/scrabble/scrabble.rb'
+
+describe Scrabble::Scrabble do
```

Comment:
@mhotan I'm not sure what you mean. So right now, the highest_score_from method handles all the cases you mentioned. For not equal in point value, it only returns the highest point scoring word. For equal in point, it checks for the length of the word and returns either the shortest or the 7 letter word. And for equal in point and length, it returns the first word as a side effect of the minmax_by method.